tests/integration/kubernetes/k8s-debug-console.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+# Copyright (c) 2025 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+load "${BATS_TEST_DIRNAME}/lib.sh"
+
+# For kata-runtime
+export KATA_HOME="/opt/kata"
+export KATA_RUNTIME="${KATA_HOME}/bin/kata-runtime"
+
+timeout=${timeout:-30}
+
+setup() {
+  if [[ "${K8S_TEST_HOST_TYPE}" != "baremetal" ]]; then
+    skip "Debug console only available from baremetal systems"
+  fi
+
+  pod_name="busybox-base-pod"
+  get_pod_config_dir
+  yaml_file="${pod_config_dir}/pod-busybox-base.yaml"
+}
+
+@test "Access and verify that the debug console is working" {
+  # Create pod
+  kubectl apply -f "${yaml_file}"
```

Comment:
We should use more of the lib.sh functions. In this case, https://github.com/kata-containers/kata-containers/blob/main/tests/integration/kubernetes/lib.sh#L83 will create the pod and wait. If it fails then it prints the pod's log and description.

Reply:
Updated to use the lib.sh method.